log/README.md

```diff
@@ -1,54 +1,45 @@
+
+
 # package log
 
-`package log` provides a minimal interface for structured logging in services.
-It may be wrapped to encode conventions, enforce type-safety, etc.
-It can be used for both typical application log events, and log-structured data streams.
+    import "/Users/harlow/code/go/src/github.com/go-kit/kit/log"
```

Comment:
Right this is because I'm running an unbuilt version of `godoc2md` from the mainline branch. That will adjust to the Repo path when I build/install it correctly.